### ISSUE TITLE
Create kit handback plan

### ISSUE THREAD

**Issue by mildlyincompetent** [srobo/tasks#115]
See also https://github.com/srobo/comp-team-coord-docs/blob/master/RoleDescriptions/kit-return.md.

**Comment by RealOrangeOne** [comment]
The list of items which we require back are documented in the rules. We should ensure teams are aware of this list.

In the past we've had reception check for items on the list, then seal the RUBs. I'm not sure how well this works with a rush of people trying to leave just after the finals

**Comment by RealOrangeOne** [comment]
We should ensure teams are made aware of the plan before the end of the competition. This reduces the chances of people needing to search for answers on Sunday afternoon, potentially holding up teams who need a quick getaway

**Comment by RealOrangeOne** [comment]
Ideally we would update the inventory as we check items in, using a bar code scanner ideally.

**Comment by PeterJCLaw** [comment]
Possibly, though that introduces a dependency on various computing things at a point where we're trying to process teams as fast as possible while also tearing down the venue. It also means training all the relevant volunteers in using the software, which is considerably more complicated than checking things off a paper list.
In the past we've done this on paper (see the docs linked in the description) and that's been good enough.

**Comment by RealOrangeOne** [comment]
The existing documentation looks to cover everything we need. We should consider this task completed as soon as https://github.com/srobo/runbook/pull/8/ merges, which confirms that the documentation exists in an accessible location.

**Comment by PeterJCLaw** [comment]
I suggest that, before closing this, we also decide:
- how many kit-return stations we run in parallel
- how we handle before-competition-end-handback (for teams leaving early [it happens], unless that's already covered)
- where we'll set these stations up (and how we do that without blocking general access to things/fire routes)
- when we'll set them up

**Comment by RealOrangeOne** [comment]
- The same number of reception channels we run
- There should be no difference between how we handle then post-competition-end
- Bar 3 could be quite a nice location to handle this on the Sunday. If not, just handling it at reception will work too. Reception should ensure that no team leaves with a kit unless they've got prior consent to keep it.
- Sunday morning. I'd be surprised if a team needs to leave on Saturday. And if so that's special case enough for us to handle it ad-hoc

**Comment by PeterJCLaw** [comment]
> * There should be no difference between how we handle then post-competition-end

I disagree. At competition-end we need to err towards processing speed. As an example if a team return a kit which is missing a piece or has something massively broken in it, then I think that before competition-end we would want to get them to search for the missing piece/understand from them why the part was broken but not reported and might spend 10 minutes doing so. At competition-end we don't have 10 minutes to spend per team and would instead need either a way to bounce them to a separate channel while we process someone else, or just judge that the loss is OK and let them go anyway.

**Comment by RealOrangeOne** [comment]
This is handled by https://github.com/srobo/runbook/pull/16 and just needs a review
